docs/reference/karmada-api/auto-scaling-resources/cron-federated-hpa-v1alpha1.md

```diff
@@ -222,7 +222,7 @@ CronFederatedHPAList contains a list of CronFederatedHPA.
 
 #### HTTP Request
 
-GET /apis/autoscaling.karmada.io/v1alpha1/namespaces/{namespace}/cronfederatedhpas/{name}
+GET /apis/autoscaling.karmada.io/v1alpha1/namespaces/\{namespace\}/cronfederatedhpas/\{name\}
```

Comment:
Given tools generate these docs, I don't know if we can update them manually. 
cc tools author here for comments. @chaosi-zju